app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -124,7 +124,7 @@ def images
         :uid_ems            => id,
         :ems_ref            => id,
         :name               => ibm_image.name,
-        :description        => ibm_image.description.presence,
+        :description        => ibm_image.specifications.image_type,
```

Comment:
Is this so that you can identify images as SAP images in provisioning later?

Reply:
Yeah, there are three image "types": `stock` (this could translate to the existing 'public' column maybe), `stock-sap` (special SAP images that you can provision with profiles to get a certified environment and better pricing) and `import` (user imported). The high level goal of this PR is to allow provisioning of these `stock-sap` images with the special profiles.